src/io/flutter/run/FlutterReloadManager.java

```diff
@@ -327,14 +319,14 @@ private boolean hasErrors(@NotNull Project project, @NotNull Module module, @Not
     }
   }
 
-  private static boolean shouldBlockReload(@NotNull DartServerData.DartError error, @NotNull Module module) {
+  private static boolean shouldBlockReload(@NotNull DartServerData.DartError error, @NonNull Project project, @Nullable Module module) {
```

Comment:
I'm assuming `NonNull` was a typo and you mean `NotNull`?

Reply:
Good catch! done